doc/doxygen/headers/glossary.h

```diff
@@ -377,7 +377,42 @@
  * parallel::distributed::Triangulation .
  * </dd>
  *
- * @see @ref boundary "The module on boundaries"
+ * @see @ref boundary "The module on boundaries".
+ *
+ *
+ * <dt class="glossary">@anchor GlossCoarseMesh <b>Coarse mesh</b></dt>
+ * <dd>
+ *   A "coarse mesh" in deal.II is a triangulation object that consists only
+ *   of cells that are not refined, i.e., a mesh in which no cell is a child
+ *   of another cell. This is generally how triangulations are first
+ *   constructed in deal.II, for example using (most of) the functions in
+ *   namespace GridGenerator, the functions in class GridIn, or directly
+ *   using the function Triangulation::create_triangulation(). One can of
+ *   course do computations on such meshes, but most of the time (see, for
+ *   example, almost any of the tutorial programs) one first refines the
+ *   coarse mesh globally (using Triangulation::refine_global()),
+ *   or adaptively (in that case first computing a refinement
+ *   criterion, then one of the functions in namespace GridRefinement,
+ *   and finally calling
+ *   Triangulation::execute_coarsening_and_refinement()). The mesh is
+ *   then no longer a "coarse mesh", but a "refined mesh".
+ *
+ *   In some contexts, we also use the phrase "the coarse mesh of a
+ *   triangulation", and by that mean that set of cells that the triangulation
+ *   started out with, i.e., from which all the currently
+ *   @ref GlossActive "active cells" of the triangulation have been obtained
+ *   by mesh refinement. (Some of the coarse mesh cells may of course also
+ *   be active if they have never been refined.)
+ *
+ *   Triangulation objects store cells in <i>levels</i>: in
+ *   particular, all cells of a coarse mesh are on level zero. Their
+ *   children (if we executed Triangulation::refine_global(1) on a
+ *   coarse mesh) would then be at level one, etc. Consequently, a
+ *   coarse mesh only has cells on level zero. The coarse mesh of a
```

Comment:
This sentence (`Consequently, a coarse mesh only has cells on level zero`) is redundant: the first sentence already says `all cells of a coarse mesh are on level zero`,